src/clipboard.ts

```diff
@@ -48,3 +48,24 @@ export const clipboardParser = new DOMParser(schema, [
   ...nodes,
   ...DOMParser.fromSchema(schema).rules,
 ])
+
+export const getListStyleType = (cssStyle: string | null) => {
```

Comment:
Doesnt look to me like something pertaining to clipboard, perhaps utils or lib folder would be more suitable for that, especially if its exported and used in some other places.